src/App.jsx

```diff
@@ -14,70 +16,71 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const getPrepareGoods = allGoods => allGoods
+  .map((good, index) => ({ good, key: index }));
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const prepareGoods = getPrepareGoods(goods);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export const App = () => {
+  const initialSelectedGood = prepareGoods[8].good;
+  const [selectGood, setSelectGood] = useState(initialSelectedGood);
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const handleProductClick = product => setSelectGood(product);
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectGood ? (
+          <>
+            <span>{`${selectGood} is selected`}</span>
             <button
-              data-cy="RemoveButton"
+              onClick={() => {
+                setSelectGood(null);
+              }}
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+            />
+          </>
+        )
+          : ('No goods selected')
+        }
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {prepareGoods.map(({ good, key }) => {
+            const isSelected = selectGood === good;
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                key={key}
+                data-cy="Good"
+                className={cn({
+                  'has-background-success-light': isSelected,
+                })}
+              >
+                <td>
+                  <button
+                    onClick={() => handleProductClick(isSelected ? null : good)}
```

Comment:
`ClearButton` should clear the selection **by setting an empty string to selectedGood**.
```suggestion
                    onClick={() => handleProductClick(isSelected ? "": good)}
```

Reply:
this is a mentor option